lib/schema.js

```diff
@@ -37,12 +44,22 @@ module.exports = [
     filters: {
       projection: {
         title: 1,
-        slug: 1,
-        type: 1,
-        tags: 1
+        _url: 1
       }
     }
   },
+  {
+    name: '_linkFile',
+    label: 'Link File',
+    type: 'joinByOne',
+    withType: 'apostrophe-file',
+    idField: 'fileId',
+    required: true,
+    projection: {
+      title: 1,
+      _url: 1
```

Comment:
Hmm, for the file this actually won't work. The actual attachment needs to be included in the projection. There isn't much in an `apostrophe-file` so I would just leave out the projection.

We should work on making sure `_url: 1` modifies its behavior to suit the type, but it doesn't handle special cases like this one yet.